UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -416,26 +416,26 @@ <h4>Examples</h4>
 								data-localization-mode="descriptive">Appearance of the text and page layout can be
 								modified according to the capabilities of the reading system (font family and font size,
 								spaces between paragraphs, sentences, words, and letters, as well as color of background
-								and text).</li>
+								and text)</li>
 							<li data-localization-id="visual-adjustments-unmodifiable"
-								data-localization-mode="descriptive">Text and page layout cannot be modified. The
+								data-localization-mode="descriptive">Text and page layout cannot be modified The
 								reading experience is close to a print version. Reading systems can still provide
-								zooming options.</li>
+								zooming options</li>
```

Comment:
This is highly problematic. Should the strings all be a single statement? Taking a period out of the middle of a paragraph can't be done, and presumably the other period here would have to be removed. Could you use commas here, perhaps, so make it more like:

> Text and page layout cannot be modified as the reading experience is close to a print version, but reading systems can still provide zooming options